web/src/js/__tests__/components/ContentView/__snapshots__/CodeEditorSpec.js.snap

```diff
@@ -0,0 +1,8 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CodeEditor Component should render correctly 1`] = `
+<div
```

Comment:
> BTW, I don't understand why the codecov coverge dropped.

Codecov bug. 

Other than that, looking at the issues you are describing the current test is good enough. I'll add a comment and we're good to go! :)